content/slides/en.yaml

```diff
@@ -78,57 +111,121 @@ levels:
+            code: |
+                {play} C4
+                {play} D4
+                {play} E4
+                {play} F4
+                {play} G4
+                {play} A4
+                {play} B4
+                {play} C5
+            notes: |-
+                Show the students how to use the `{play}` command in Hedy. This is also a good time to review the classroom rules about headphone usage.
+                The students can now continue with the music adventure.
+        11:
             header: 'Drawing with the turtle: {forward}'
             text: '`{forward}` is used to move the turtle forward.'
             code: '{forward} 50'
-        11:
+            notes: |-
+                Now that we have seen at text codes, we will now look at drawing codes next. Show the students how to use the forward command.
+        12:
+            header: 'Drawing with the turtle: {forward}'
+            text: 'How do we go backwards?'
+            code: '{forward} -50'
+            notes: |-
+                Ask the students if they have any idea how to go backwards. The answer is using a negative number, like -50.
+        13:
             header: 'Drawing with the turtle: {turn}'
             text: '`{turn}` is used to make the turtle turn left or right.'
             code: |-
                 {forward} 50
                 {turn} {left}
-        12:
+        14:
             header: Debugging
-            text: Debugging a code means getting rid of mistakes in the code. Let's debug this code together. Click the ladybug button to start the debugger.
+            text: Using the debugger when drawing
             code: |-
-                I am DJ Hedy
-                {print} What song would you like to hear?
-                {ask} I like that song too!
-                {print} Next up... {echo}
+                {forward} 50
+                {turn} {left}
+                {forward} 100
+                {turn} {left}
+                {forward} 50
+                {turn} {left}
             debug: 'True'
-        13:
-            header: Let the programming fun begin!
-            text: Enjoy the adventures in level 1!
+            notes: |-
+                The debugger can come in handy when making a drawing. The students can now continue with the rest of the adventures of this level.
+        15:
+            header: Let's review!
+            text: What did you learn today?
+            notes: |-
+                Ask the students to list what they've learned today.
+        16:
+            header: Let's review!
+            text: |-
+                Which adventure did you like best? Why?
+
+                Which adventure did you like least? Why?
+
+            notes: |-
+                Ask the students for their opinion.
+        17:
+            header: Show and tell!
+            text: Would anyone like to show the class a program they've made?
+            notes: |-
+                Let some students show their work to the class.
```

Comment:
Would it be helpful to give a note on how to do that (if students in a class on the class page you can show adventures of students)? Maybe that is too much and we don't want that kind of information on different places.